### ISSUE TITLE
Correct entry for Germany

### ISSUE THREAD

**Issue by yatil** [w3c/wai-policies-prototype#247]
The page says BITV is an “Accessibility law,” but to be correct, it should be an “Ordinance.” I think to simplify the interface; we could just use “Policy” which sounds very similar (to me).

(No, I don’t now why I didn’t see it before.)

/via @tcaspers on twitter: https://twitter.com/tcaspers/status/872743409550753794

**Comment by yatil** [comment]
@maryjom Can you indicate if that is OK or if we need the “Ordinance” category… I am happy to make the changes once confirmed.

**Comment by maryjom** [comment]
I guess I don't understand.  What is an "ordinance"?

**Comment by tcaspers** [comment]
@maryjom An ordinance is a kind of a statutory instrument one or two notches below a proper law that (German?) gov’t can use to issue a legally binding decree. But then again, I’d be fine with policy as well…

**Comment by yatil** [comment]
I have fixed the issue in Pull Request #258. @maryjom can you review it to make sure I didn’t screw up the nomenclature ;-)
